### ISSUE TITLE
Myrmex and Dragon Pathfinding Megaissue

### ISSUE THREAD

**Issue by AlexModGuy** [Alex-the-666/Ice_and_Fire#1449]

**Comment by AlexModGuy** [comment]
should be drastically reduced in 1.8.0

**Comment by darkshadow543** [comment]
Other things I noticed that are problems with myrmex pathfinding

- Queen likes to climb walls, which causes eggs to drop to the floor and become items which then despawn.  This means colonies struggle to grow.

-Workers like to walk in circles or move in a circular way toward thier target and have a tendency to forget there are blocks over thier head, causing them to get stuck.

-Sentinels will walk straight into walls and suffocate, I observed this twice from the same sentinel, and the wall was one wide.  They also circle dance, sometimes on high blocks and fall to death.  

These are some pathing issues I noticed in version 1.8.0
